### ISSUE TITLE
Widget for quick note taking (intray)

### ISSUE THREAD

**Issue by Asalle** [orgzly/orgzly-android#740]
- I have searched for existing issues that may be the same as or related to mine.

If you're familiar with GTD, Allen uses a notebook called "intray" for all the incoming things, brainstorming etc. Basically this is the entry point for any note that gets later sorted into other notebooks. It's very handy and I use it all the time.
But it gets a bit old to open the app, open the needed notebook (intray.org) and click "Add one below" and then type my quick note.
Can we add a widget for adding new notes to a selected notebook so this has less friction?
Thank you for an amazing app! :pray:

**Comment by mroman42** [comment]
I think this can already be done, right? There is a shortcut called "New note". When you drop the shortcut, it should ask you to pick a notebook; once you pick the notebook, you can use the shortcut to start adding a note to that specific notebook with a single tap.

**Comment by Asalle** [comment]
My idea was to maybe make it even more frictionless and make a widget that
will only add notes to a single (selected ahead of time) notebook. So that
the user could just type the note and press ✅

**Comment by Asalle** [comment]
Alternatively, we can make the existing widget pick up the "default notebook" that you can set in the preferences.

**Comment by Asalle** [comment]
Alright, just tested, it does pick up the default notebook from the widget (the trick was to specify the name of the notebook in the settings, but without the .org). I'm closing the issue. Thank you for the support! 🙏

**Comment by mroman42** [comment]
Happy you found a solution :) Thanks go to the maintainers.

Maybe I am missing something, but I think that the widget that adds to a single (selected ahead of time, not necessarily the default) notebook already works as well. I have different widgets on my home screen for different notebooks, and I picked each of these notebooks when I added the corresponding widgets to the home screen. Maybe that is closer to what you had in mind (?). I add a screenshot for reference, in case someone finds the issue but does not want to only use the default notebook.

![photo5796681055061783292](https://user-images.githubusercontent.com/5337877/87394980-d82d8800-c5b0-11ea-8b0e-71af721b2dd5.jpg)

Oh, now I think I am seeing the problem. If I add the widget by pressing the orgzly icon, then it will **not** ask me for the notebook.  To get these widgets to ask me for the specific notebook, I had to create them by pressing on an empty space of the screen and then add then through the "widgets" menu.

In other words, depending on how the widget is added, it will ask the user to pick a notebook or not. I am not sure if this was intended, but it can be very confusing.
